service/core/client/src/main/scala/com/lightbend/lagom/internal/client/ClientServiceCallInvoker.scala

```diff
@@ -175,17 +180,22 @@ private[lagom] abstract class ClientServiceCallInvoker[Request, Response](
   /**
    * A call that is strict in both directions.
    */
-  private def makeStrictCall(requestHeader: RequestHeader, requestSerializer: MessageSerializer[Request, ByteString],
-                             responseSerializer: MessageSerializer[Response, ByteString], request: Request): Future[(ResponseHeader, Response)] = {
+  private def makeStrictCall(
+    requestHeader: RequestHeader,
+    requestSerializer: MessageSerializer[Request, ByteString],
+    responseSerializer: MessageSerializer[Response, ByteString],
+    request: Request
+  ): Future[(ResponseHeader, Response)] = {
 
     val headerFilter = descriptorHeaderFilter(descriptor)
     val transportRequestHeader = headerFilterTransformClientRequest(headerFilter, requestHeader)
```

Comment:
The original `requestHeader` is also used in one other place further down:

```scala
      val responseHeader = headerFilterTransformClientResponse(headerFilter, transportResponseHeader, requestHeader)
```

It occurred to me that it would be easier to avoid this mistake if these methods were refactored so the original `requestHeader` isn't in scope. So, one method takes the original request, immediately transforms it, then calls another method, passing the transformed request without the original.

Reply:
Sure, I will refactor them.

Strange enough, I thought I have changed the one you pointed out. 🤷‍♂️